config/v3_4_experimental/translate/translate.go

```diff
@@ -27,9 +28,36 @@ func translateIgnition(old old_types.Ignition) (ret types.Ignition) {
 	return
 }
 
+func translateFileEmbedded1(old old_types.FileEmbedded1) (ret types.FileEmbedded1) {
+	tr := translate.NewTranslator()
+	tr.Translate(&old.Append, &ret.Append)
+	tr.Translate(&old.Contents, &ret.Contents)
+	if old.Mode != nil {
+		// We're planning to support the special mode bits for specs >=3.4.0,
```

Comment:
`planning to` implies that we're doing it in the future.  Maybe "We support the special mode bits only in specs >= 3.4.0"?